ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1078,14 +1099,20 @@ static PrjFS_Result HandleFileNotification(
     const MessageHeader* request,
     const char* relativePath,
     const char* absolutePath,
+    const char* relativeFromPath,
     bool isDirectory,
     PrjFS_NotificationType notificationType)
 {
+    // Don't send null to providers
+    relativePath = relativePath != nullptr ? relativePath : "";
```

Comment:
@wilbaker what are your thoughts on having the check here.  Option 2 would be send "" instead of nullptr in the TrySendMessage many levels up.  Both approachs have pros/cons.  Since we were already doing the nullptr I stuck with it for now.

Reply:
I'll throw an option 3 out there as well, making the check in the hardlink case of `OnNotifyOperation`